Challenge2/pricingCard.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="pricingCard.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <center>
+      <div class="current-plan">
+        <span>Your current plan:</span>
+        <br />
+        <span class="plan-name"> Starter Trial • 500MAUs </span>
+      </div>
+    </center>
+    <center>
+      <div class="bill-type">
+        <h1 class="page-title">Choose a plan</h1>
+        <br />
+        <label for="monthly-plan">
+          <input
+            type="radio"
+            name="subscription-type"
+            id="monthly-plan"
+          />Billed monthly
+        </label>
+        <label for="anually-plan">
+          <input
+            type="radio"
+            name="subscription-type"
+            id="anually-plan"
+            checked
+          />Billed anually
+        </label>
+      </div>
+    </center>
+    <!--<div class="monthly">Billed monthly</div>
```

Comment:
No commented code should be pushed. Please check before you push your code.
Remove comments.